### PR TITLE
Feature/ode 115 create feature posts of blog

### PR DESCRIPTION
**Pull request by caovv62**
create feature posts

### CODE REVIEW

**Comment by ngoc-nguyenminh** [comment]
You should make 1 pull request for 1 task
it doesn't work, on both PC and mobile

File: resources/js/carousel.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+document.addEventListener('DOMContentLoaded',function(){
+    var carouselSlide = document.querySelector('.Card_List');
+    var carouselImg = document.querySelectorAll('.contentCard');
+
+    //
```
Comment (ngoc-nguyenminh) [inline resources/js/carousel.js:5]: delete unnecessary lines

File: resources/js/carousel.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    var prevBtn = document.querySelector('#prevBtn');
+    var nextBtn = document.querySelector('#nextBtn');
+
+    //
```
Comment (ngoc-nguyenminh) [inline resources/js/carousel.js:10]: here too

File: resources/js/carousel.js (hunk 1 above)
Comment (caovv62) [reply resources/js/carousel.js:5]: các dòng "//" à e?

**Comment by caovv62** [comment]
@ngoc-nguyenminh I completed task work on moblie. Do you can review it?

**Pull request closed without merging**